src/views/RecIM/components/Forms/CreateTeamForm/index.js

```diff
@@ -122,9 +122,9 @@ const CreateTeamForm = ({
 
     let teamCreationRequest = { ...currentInfo, ...newInfo };
```

Comment:
I'm not sure "...Request" is the best terminology here. A request is a different thing than the object to upload. Perhaps `teamUpload` or `newTeam`.

Reply:
I think the Request terminology was originally used since it was the passed/requested object to the route. The naming was inherited from the AlumniUpdateForm earlier this summer

Reply:
But AlumniUpdateForm was actually sending an information update request to development. This is actually going to make changes in the database. The issue is kind of a result of the overloaded term, 'request'. (http request vs request for info update)

Reply:
I see, I'll make a note for the team to make an update on that front.